landoapi/api/landings.py

```diff
@@ -71,3 +69,25 @@ def get(landing_id):
         )
 
     return landing.serialize(), 200
+
+
+def update(landing_id, data):
+    """Update landing on pingback from Transplant."""
+    try:
+        landing = Landing.query.filter_by(
+            id=landing_id, request_id=data['request_id']
+        ).one()
+    except NoResultFound:
+        return problem(
+            404,
+            'Landing not found',
+            'The requested Landing does not exist',
+            type='https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404'
+        )
+
+    landing.error = data.get('error_msg', '')
+    landing.result = data.get('result', '')
+    landing.status = TRANSPLANT_JOB_LANDED if data['landed'
```

Comment:
I'm adding description of fields which are received from Transplant (from Google doc)